headers/types/dungeon_mode/dungeon.h

```diff
@@ -1691,13 +1691,12 @@ struct dungeon {
     undefined field_0x2caf5;
     undefined field_0x2caf6;
     undefined field_0x2caf7;
-    // 0x2CAF8: Related to playing dungeon music? Initialized to 0x4.
+    // 0x2CAF8: Related to playing dungeon music? Initialized to 0x0 at the
```

Comment:
This used to say initialized to 0x4, now it says 0x0. Is this a correction or a mistake?

Reply:
This is a correction.